pkg/txn/storage/txn/mem_handler.go

```diff
@@ -967,11 +968,19 @@ func (m *MemHandler) rangeBatchPhysicalRows(
 		}
 
 		// add version
-		a := rand.Int63()
-		b := rand.Int63()
-		version := types.Decimal128FromInt64Raw(a, b)
+		var a int64
```

Comment:
This is totally broken -- we need to switch rowid from decimal to [16]byte.

There is no guanrantee that a decimal128 can be converted to string (NaN, for example), and there is no guanrantee that when you read back from string, the decimal bits will stay the same (cohort).

Reply:
I agree.
The TAE and execution codes rely on this decision: https://github.com/matrixorigin/matrixone/blob/e1fd77f530f8e4b93a5179b7c71315fe265f9f54/pkg/sql/colexec/update/update.go#L145
I'll fix this after TAE and executor fixes.